frontend/src/pages/impressum/components/DeveloperWrapItem.tsx

```diff
@@ -0,0 +1,26 @@
+import { Flex, HStack, Tag, Text, Image, useColorModeValue, WrapItem } from '@chakra-ui/react'
+import { API_BASE_URL } from '../../../util/configs/environment.config'
+import { customTheme } from '../../../util/configs/theme.config'
+import { Dev } from '../../../api/hooks/useDevelopers'
+
+type Props = {
+  dev: Dev
+}
+
+export const DeveloperWrapItem = ({ dev: { name, img, tags } }: Props) => {
```

Comment:
I needed to rename this, because the git history did not like me changing a lowercase letter to an upper one.